lnwallet/chainfee/estimator.go

```diff
@@ -642,30 +642,37 @@ type SparseConfFeeSource struct {
 	URL string
 }
 
+// WebAPIResponse is the response returned by the fee estimation API.
+type WebAPIResponse struct {
+	// FeeByBlockTarget is a map of confirmation targets to sat/kvb fees.
+	FeeByBlockTarget map[uint32]uint32 `json:"fee_by_block_target"`
+
+	// MinRelayFeerate is the minimum relay fee in sat/kvb.
+	MinRelayFeerate SatPerKVByte `json:"min_relay_feerate"`
+}
+
 // parseResponse attempts to parse the body of the response generated by the
 // above query URL. Typically this will be JSON, but the specifics are left to
 // the WebAPIFeeSource implementation.
 func (s SparseConfFeeSource) parseResponse(r io.Reader) (
-	map[uint32]uint32, error) {
-
-	type jsonResp struct {
-		FeeByBlockTarget map[uint32]uint32 `json:"fee_by_block_target"`
-	}
+	*WebAPIResponse, error) {
 
-	resp := jsonResp{
+	resp := &WebAPIResponse{
 		FeeByBlockTarget: make(map[uint32]uint32),
+		MinRelayFeerate:  0,
```

Comment:
Do we want this to be 1 since that's the actual default min relay fee rate?

Reply:
yeah this just inits the struct - if we don't do it the default empty value will be used, but I always prefer explicitly defining it first.

Reply:
Yeah that's fine by me. I definitely like the explicit initialization. I've been bitten by implicit zero-values more times than I can count.